src/app/content/components/Sidebar.tsx

```diff
@@ -4,131 +4,56 @@ import React, { Component, ComponentType } from 'react';
-const sidebarClosedOffset = sidebarOpenWidth - sidebarClosedWidth;
+export const sidebarWidth = 33.5;
 const sidebarTransitionTime = 300;
-const sidebarControlSize = 40;
-const sidebarPadding = 10;
+const sidebarPadding = 1;
 
-const SidebarPlaceholder = styled.div<{isOpen: boolean}>`
-  overflow-x: hidden;
-  transition: all ${sidebarTransitionTime}ms;
-
-  ${(props) => !props.isOpen && css`
-    background-color: #ccc;
-    min-width: ${sidebarClosedWidth}px;
-  `}
-  ${(props) => props.isOpen && css`
-    background-color: white;
-    min-width: ${sidebarOpenWidth}px;
-  `}
-`;
-
-type StyledSidebarControlProps = React.HTMLProps<HTMLButtonElement> & {
-  isOpen: boolean,
-};
-
-const SidebarControl = styled(({isOpen, ...props}: StyledSidebarControlProps) =>
-  <FormattedMessage id={isOpen ? 'i18n:toc:toggle:opened' : 'i18n:toc:toggle:closed'}>
-    {(msg: Element | string) => {
-      const txt = assertString(msg, 'Aria label only supports strings');
-      return <button {...props}
-          aria-label={txt}
-        >
-        <span></span>
-        <span></span>
-        <span></span>
-      </button>;
-    }}
-  </FormattedMessage>
-)`
-  position: fixed;
-  height: ${sidebarControlSize}px;
-  width: ${sidebarControlSize}px;
-  margin-top: ${sidebarPadding}px;
-  background: none;
-  padding: 0;
-  border: none;
-  z-index: 1;
-  outline: none;
-  transition: all ${sidebarTransitionTime}ms;
-  transform: translateX(0px);
-
-  span {
-    width: 100%;
-    background-color: #000;
-    padding-top: 10%;
-    display: block;
-    visibility: visible;
-    opacity: 1;
-    transition: all ${sidebarTransitionTime}ms;
-  }
-
-  ${(props) => !props.isOpen && css`
-    left: ${sidebarOpenWidth - sidebarControlSize - sidebarPadding}px;
-
-    transform: translateX(-${sidebarClosedOffset}px);
-    span {
-      margin-top: 15%;
-    }
-  `}
-
-  ${(props) => props.isOpen && css`
-    // extra padding on the left for scroll bars when open
-    left: ${sidebarOpenWidth - sidebarControlSize - sidebarPadding - 10}px;
-
-    span:nth-child(1) {
-      transform: rotateZ(45deg);
-    }
-    span:nth-child(2) {
-      opacity: 0;
-    }
-    span:nth-child(3) {
-      transform: rotateZ(-45deg);
-      margin-top: -20%
-    }
-  `}
-`;
+const sidebarPositionBreak = '64em';
 
 const SidebarBody = styled.div<{isOpen: boolean}>`
-  position: fixed;
-  left: 0;
-  height: 100%;
-  padding-top: ${sidebarPadding}px;
-  width: ${sidebarOpenWidth}px;
+  top: 0;
+  height: 100vh;
   overflow-y: auto;
-  font-size: 15px;
-  padding: ${sidebarPadding}px;
+  padding: ${sidebarPadding}rem;
   transition: all ${sidebarTransitionTime}ms;
-  transform: translateX(0px);
+  background-color: ${theme.color.neutral.darker};
 
   ol {
     padding-inline-start: 10px;
   }
 
   > * {
-    transition-property: visibility;
-    transition-delay: 0s;
+    transition: all ${sidebarTransitionTime}ms;
     visibility: visible;
+    opacity: 1;
+  }
+
+  width: 33.5rem;
```

Comment:
move this to a constant, note that this is the large width, there will be a small one too soon